Integrations/integration-ServiceNow.yml

```diff
@@ -1303,7 +1331,7 @@ script:
     arguments:
     - name: id
       default: true
-      description: Ticket System ID to retrieve
```

Comment:
Why did you remove `to retrieve`?

Reply:
to be consistent with the other commands